snapcraft/internal/lxd/_containerbuild.py

```diff
@@ -138,6 +139,33 @@ def _configure_container(self):
         subprocess.check_call([
             'lxc', 'config', 'set', self._container_name,
             'environment.LC_ALL', 'C.UTF-8'])
+        self._set_image_info_env_var()
+
+    def _set_image_info_env_var(self):
+        FAILURE_WARNING_FORMAT = (
+            'Failed to get container image info: {}\n'
+            'It will not be recorded in manifest.')
```

Comment:
in *the* manifest
or _it will not be recorded as part of the manifest_

Reply:
one of the recommendations from mtp was to not use unnecessary articles.